crates/ibc/src/core/ics03_connection/handler/conn_open_init.rs

```diff
@@ -118,6 +120,8 @@ pub(crate) fn process(
         msg.client_id_on_a.clone(),
         Counterparty::new(
             msg.counterparty.client_id().clone(),
+            // Note: the counterparty connection Id passed by the relayer alternated to `None`,
```

Comment:
Same, I don't think we need this comment

Reply:
[e2ea605](https://github.com/cosmos/ibc-rs/pull/360/commits/e2ea6054fee2be83cd9421fce8b0b21bd92e754b)